docs/debugging.md

```diff
@@ -1,5 +1,11 @@
 # Debugging Guide
 
+## Logo for my platform is not appearing on the Home Page or search results
```

Comment:
this shouldn't be the very first item - the quickstart debugging docs are much more important

Reply:
Did not realise this was only for quickstart debugging. Changed the name, moved that FAQ to it's own section.

Reply:
It doesn't need to be exclusive to quickstart - my comment was that this new section should be at the bottom of the doc